sktime/transformations/bootstrap/_tsbootstrap.py

```diff
@@ -0,0 +1,135 @@
+"""Bootstrapping adapter for tsbootstrap."""
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["benheid"]
+
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+
+
+class TSBootstrapAdapter(BaseTransformer):
+    """Adapter for TSBBootstrap.
+
+    The bootstrap samples will be returned as a Panel with the first level
+    being integer index of the synthetic sample.
+
+    For hierarchical data, the bootstrap index will be added as the index
+    at integer position -2.
+
+    Parameters
+    ----------
+    bootstrap : bootstrap from tsbootstrap
+        default = SlidingWindowSplitter(window_length=3, step_length=1)
```

Comment:
The signature of `__init__` has no default value.

Reply:
Done